documentation/faq/70-packages.md

```diff
@@ -2,6 +2,19 @@
 question: How do I fix the error I'm getting trying to include a package?
 ---
 
-Most of these issues come from Vite trying to deal with non-ESM libraries. You may find helpful examples in [the Vite issue tracker](https://github.com/vitejs/vite/issues). The most common solutions would be to try moving the package between `dependencies` and `devDependencies` or trying to `include` or `exclude` it in `optimizeDeps`. Packages which use `exports` instead of `module.exports` are currently failing due to a [known Vite issue](https://github.com/vitejs/vite/issues/2579). You should also consider asking the library author to distribute an ESM version of their package or even converting the source for the package entirely to ESM.
+If youre seeing errors like:
+
+* `[vite] Error when evaluating SSR module /node_modules/....`
```

Comment:
I think if we add error messages we should try to match them with the appropriate solution. I'm not actually sure how you fix this error and whether any of the solutions below work for it

Also, I'd remove the `/node_modules/....` because you get `C:\` on Windows

Reply:
From what I've seen this is the error fixed by moving to devDependencies, are you saying co-locate those two?

Reply:
yeah

Reply:
I'm not sure how this should be done since a few of the solutions would work for these errors, hence why I put them at the top.

What other errors are people seeing that I should put in here and how would you co-locate the errors when multiple solutions could solve the issue?

Reply:
I don't know. I'm pretty hesitant about this PR to be honest because I don't think we have a great understanding of the error messages and solutions, etc. I think most of the error messages can be found in the issue tracker and I've been pointing people to https://kit.svelte.dev/faq in those issues, so I think that this FAQ should already be fairly discoverable from the error message. Really I'd probably rather spend time trying to get https://github.com/sveltejs/kit/issues/1016 in than improving the docs around this so that people don't hit as many of these issues in the first place and so that they're less complicated when they are encountered

Reply:
I agree fixing the underlying issue is superior to docs about the behavior, was just trying to help people out until the fix 🤷